lib/loader.rb

```diff
@@ -1,4 +1,23 @@
+require 'sequel'
+
+DB = Sequel.connect(
+  adapter: 'mysql2',
+  host: ENV.fetch('DB_HOSTNAME'),
+  database: ENV.fetch('DB_NAME'),
+  user: ENV.fetch('DB_USER'),
+  password: ENV.fetch('DB_PASS')
+)
+
+module Common;
+end
+
 module PerformancePlatform
   module Gateway; end
   module Repository; end
+  module UseCase; end
+  module Presenter; end
 end
+
+require 'require_all'
```

Comment:
I'd pull this to the top of the file - surprised linter isn't complaining. Maybe it isn't in this project?

Reply:
Makes sense